components/src/components/molecules/Dropdown/Dropdown.tsx

```diff
@@ -556,6 +567,7 @@ export const Dropdown = ({
         setIsOpen={setIsOpen}
         shortThrow={shortThrow}
         width={
+          inner &&
```

Comment:
maybe have this be a prop like `inheritContentWidth` instead of disabling for all but inner dropdowns

Reply:
done.